src/hotspot/cpu/x86/x86.ad

```diff
@@ -3673,6 +3681,16 @@ instruct convF2HF_mem_reg(memory mem, regF src, kReg ktmp, rRegI rtmp) %{
   ins_pipe( pipe_slow );
 %}
 
+instruct vconvF2HF(vec dst, vec src) %{
```

Comment:
We do have a destination memory flavour of VCVTPS2PH, adding a memory pattern will fold subsequent store in one instruction.